libafl/src/feedbacks/stdio.rs

```diff
@@ -35,6 +36,34 @@ pub struct StdOutToMetadataFeedback {
     o_ref: Handle<StdOutObserver>,
 }
 
+impl StdOutToMetadataFeedback {
+    /// Append to the testcase the generated metadata in case of a new corpus item.
+    #[inline]
+    fn append_stdout_observation_to_testcase<S, OT>(
+        &mut self,
+        observers: &OT,
+        testcase: &mut Testcase<S::Input>,
+    ) -> Result<(), Error>
+    where
+        S: State + UsesInput,
```

Comment:
Can you try to not use UsesInput here?
You can specify
S: HasCorpus and make the input to be <S::Corpus as Corpus>::Input